Source/Core/Core/Boot/Boot.cpp

```diff
@@ -273,14 +271,6 @@ std::unique_ptr<BootParameters> BootParameters::GenerateFromFile(std::vector<std
                                               std::move(boot_session_data_));
     }
 
-    if (is_drive)
-    {
-      PanicAlertFmtT("Could not read \"{0}\". "
-                     "There is no disc in the drive or it is not a GameCube/Wii backup. "
-                     "Please note that Dolphin cannot play games directly from the original "
-                     "GameCube and Wii discs.",
-                     path);
-    }
     else
```

Comment:
You should probably get rid of this `else` and only leave its contents. GitHub won't let me add a working suggestion tho; so remove the `else` and the curlies, but leave the `PanicAlertFmtT`.

Reply:
Done!